Resources/Prototypes/Entities/Objects/Weapons/Guns/SMGs/smgs.yml

```
@@ -228,7 +228,7 @@
   name: WT550
   parent: [ BaseWeaponSubMachineGun, BaseRestrictedContraband ]
   id: WeaponSubMachineGunWt550
-  description: An excellent SMG, produced by NanoTrasen's Small Arms Division. Uses .35 auto ammo.
+  description: A truly unique firearm, the WT550 was created by NanoTrasen's Small Arms Division as an SMG fully controllable with one hand. It would likely have fully replaced the Drozd, but high manufacturing costs forced NT to scale back production. It contains an exotic internal recoil buffer and feeds from special 30-round top-mounted .38 magazines.
```

Unnecessary RP explanation. Plus you can just buy these on the cheap so it doesn't make sense.

```suggestion
  description: A truly unique firearm, the WT550 was created by NanoTrasen's Small Arms Division as an SMG fully controllable with one hand. Feeds from special 30-round top-mounted .38 magazines.
```